src/main/java/mate/academy/hibernate/relations/exceptions/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.hibernate.relations.exceptions;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message, Exception e) {
+        super(message);
```

Comment:
To preserve the original exception's stack trace and provide more context, pass the original exception `e` as a cause to the superclass constructor: `super(message, e);`.